code/modules/mob/living/carbon/human/human.dm

```diff
@@ -159,7 +159,7 @@
 		if(!prob(martial_art.deflection_chance))
 			return ..()
 		if(!src.lying && dna && !dna.check_mutation(HULK)) //But only if they're not lying down, and hulks can't do it
-			src.visible_message("<span class='warning'>[src] deflects the projectile!</span>", "<span class='userdanger'>You deflect the projectile!</span>")
+			src.visible_message("<span class='danger'>[src] deflects the projectile, there's no way you'll be able to hit them with ranged weapons!</span>", "<span class='userdanger'>You deflect the projectile!</span>")

```

Comment:
We've done "not on the nose" for a long time. I was hesitant too but then I was flooded with 1,000 flashbacks of officers continually firing energy weapons into double eswords/eshields/etc. and I resolved that I wasn't going to continue the legacy of the game not telling you when you're shooting at something that can't be shot.